### DISCUSSION TITLE
Do not show properties with inaccesible access methods

### DISCUSSION THREAD

**Discussion by DjArt** [dotnet/csharplang#7774] [Language Ideas] [answered]
When property defined like
```C#
public Predicate<T>? Selector { protected get; init; }
```
or even without `get` method, it shouldn't show by IDE in cases where you can't call these accessors.

**Comment by HaloFour** [comment, marked as answer]
That sounds like a tooling problem, I'd open the issue on the [Roslyn](https://github.com/dotnet/roslyn) repository.

**Comment by CyrusNajmabadi** [comment]
Closing out. This is not a language issue.
